File: javascript/6kyu - B.js
//Link -> https://www.codewars.com/kata/523f5d21c841566fde000009

function arrayDiff(a, b) {
    var distinctArray = []

    a.forEach(element => {
        if(b.includes(element) == false){
            distinctArray.push(element)
        }
    });

    return distinctArray
}

File: kotlin/6kyu - B.kt
//Link -> https://www.codewars.com/kata/587309155cfd6b9fb60000a0

package com.codewars.geoffp

object UnwantedDollars {
  fun moneyValue(money:String):Double {
    var moneyString = money.replace(" ","").replace("$", "")
    var monetaryValue = 0.0

    if(!moneyString.contains('.')){
        moneyString = "$moneyString.0"
    }

    if(moneyString.indexOf('.') == moneyString.length - 2){
        moneyString += "0"
    }

    monetaryValue = moneyString.toDouble()

    return monetaryValue
  }
}
